bundle_validator/README.md

```diff
@@ -0,0 +1,37 @@
+# Validate STIX bundle
+
+You can easily validate your stix bundle file by following the below steps:
+
+## Prerequisites:
+
+The following needs to be installed on your local machine:
+
+1. Python 3
+2. GIT
+
+
+If you have not already cloned the [stix-shifter github project](https://github.com/opencybersecurityalliance/stix-shifter): 
+```
+git clone https://github.com/opencybersecurityalliance/stix-shifter.git
+```
+
+## Validator Usage:
+
+1. Open a terminal 
+2. cd into /stix-shifter/bundle_validator
+3. Copy your STIX bundle JSON file into the bundle_validator directory
+4. Run the validate.sh script. You need to specify the bundle json file name:
+    ```
+    ./validate.sh <STIX Bundle JSON FIle>
+    ```
+5. After successfull validation, you should see a messegae in your terminal: `STIX Bundle validated!!`
+6. For unsuccessfull validation, you should see mainly two types of error-
+    1. If JSON format is invalid: `Malformed JSON in the STIX Bundle: <ERROR details>`
+    2. If the file contains invalid STIX Objects, you should see errors/warnings with heading `[X] STIX JSON: Invalid`. It is mandatory to fix the errors red marked as `[X]`. Warnings which are yellow marked as `[!]`, can be ingnored but recommended to fix. For example-
```

Comment:
change to "marked red" and "marked yellow"

Reply:
done

Reply:
missed changing to "marked yellow".

Reply:
I thought I changed it . 
```
2. If the file contains invalid STIX Objects, you should see errors/warnings with heading `[X] STIX JSON: Invalid`. It is mandatory to fix the errors marked red as `[X]`. Warnings which are marked yellow as `[!]`, can be ingnored but recommended to fix. For example-
```